nixos/modules/config/users-groups.nix

```diff
@@ -574,6 +574,9 @@ in {
     # for backwards compatibility
     system.activationScripts.groups = stringAfter [ "users" ] "";
 
+    # Used in nixos-install
+    system.build.users-groups-spec = spec;
```

Comment:
I am afraid that I implemented this `build` option and it isn't clear to me what is happening here. Can you help?

Reply:
This attribute is used later in `nixos/modules/system/activation/top-level.nix` to link `users-groups.json` file to `/nix/var/nix/profiles/system/users-groups.json`